core/tests/test_utils.py

```diff
@@ -143,16 +148,39 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Returns a string for unicode compatibility test
+
+        Args:
+            suffix: string . Test string appended to unicode string
+
+        Returns:
+            string . Unicode test string
```

Comment:
Again, no need of a space between 'string' and '.'
Same goes for all the other places, no need to put a space between period(.) and the previous word.